e2e/go.mod

```diff
@@ -241,4 +241,4 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 
 //replace github.com/strangelove-ventures/interchaintest/v8 => github.com/notional-labs/interchaintest/v8 v8.0.0-20230905211440-c2dc240d2221
 
-replace github.com/strangelove-ventures/interchaintest/v8 => github.com/colin-axner/interchaintest/v8 v8.0.0-20230906142353-5d89200aad4a
+replace github.com/strangelove-ventures/interchaintest/v8 => github.com/crodriguezvega/interchaintest/v8 v8.0.0-20230912182717-55add1fb6aff
```

Comment:
I had to fork Colin's fork to add the extra `ResolveDenom` flag in the request.

Reply:
Can we directly the bank grpc in the test suite (add it if it doesn't exist). It can be added in the grpc.go file, this is what that function is [doing](https://github.com/colin-axner/interchaintest/pull/1/files#diff-3ca0c69f200a759782b915d3da5fbf5f0c5cbe3507e323bd790771e0e05b7f83R568)

Reply:
I think I'd prefer to use interchaintest function, so that we don't duplicate that code. We already use interchaintest's `GetBalance` function as well. If we merge [this PR](https://github.com/colin-axner/interchaintest/pull/1) into your fork, then we can keep using your fork, and wait for interchaintest to merge your PR.

Reply:
I'm not sure I see the benefit of interchaintest defining grpc's that are exposed to users like us. It's a very low lift to define the grpc's we use and the parameters we use for a request. This pattern has become [well defined](https://github.com/cosmos/ibc-go/pull/4642#discussion_r13254465340) and I think has worked really well thus far. I think limiting our dependencies on external libraries is useful as it gives us more granular control over breaking changes/stability and this seems like a worth while trade off in terms of code duplication

Reply:
All righty. I've made the change and opened https://github.com/cosmos/ibc-go/issues/4661 to migrate the calls of `GetBalance` as well.

Reply:
Thank you!